ouroboros-consensus/src/Ouroboros/Consensus/NodeKernel.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE FlexibleContexts      #-}
 {-# LANGUAGE NamedFieldPuns        #-}
+{-# LANGUAGE PatternGuards         #-}
```

Comment:
Is this pragma actually needed? I have been writing pattern guards without it.

Reply:
Ah; it seems they are part of Haskell2010: http://www.haskell.org/onlinereport/haskell2010/haskellch3.html#x8-460003.13

Reply:
Fixed, thanks.